pom.xml

```diff
@@ -473,8 +473,8 @@
             <configuration>
               <rules>
                 <requireMavenVersion>
-                  <version>[3.3.1,)</version>
-                  <message>3.3.1 required to at least look at .mvn/ if it exists.</message>
+                  <version>[3.8.1,)</version>
+                  <message>3.8.1 required to no longer download dependencies via HTTP (use HTTPS instead).</message>
```

Comment:
no problems with it if we can get the infra upgraded, should be straight forward 😄

Reply:
Probably https://github.com/jenkins-infra/docker-inbound-agents/blob/474d107b91f338f88611a0688b30ab81c0f0f9d9/maven/jdk11/Dockerfile#L3 ?

Reply:
also https://github.com/jenkins-infra/packer-images/blob/main/.auto.pkrvars.hcl#L1 and https://github.com/jenkins-infra/docker-inbound-agents/blob/474d107b91f338f88611a0688b30ab81c0f0f9d9/maven/jdk8/Dockerfile#L3

Reply:
cc @dduportal / @olblak if you have time to help

Reply:
Sounds good, updating the ACI images is an easy one (built daily), for the AMI it needs a manual operation on ci.jenkins.io that should be advertised in advance to avoid breaking stuff.

Reply:
Shall we add addition templates at first or replacing the old ones?

Reply:
Just replace old I think?

Reply:
Good, then:
- https://github.com/jenkins-infra/packer-images/pull/68
- https://github.com/jenkins-infra/docker-inbound-agents/pull/4
=> I'll plan the change for this friday

Reply:
```shell
15:18:07  + mvn -v
15:18:08  Apache Maven 3.8.1 (05c21c65bdfed0f71a2f2ada8b84da59348c4c5d)
15:18:08  Maven home: /usr/share/maven
15:18:08  Java version: 11.0.11, vendor: Oracle Corporation, runtime: /usr/local/openjdk-11
15:18:08  Default locale: en, platform encoding: UTF-8
15:18:08  OS name: "linux", version: "5.4.81-microsoft-standard", arch: "amd64", family: "unix"
```
=> ACI OK (ref https://ci.jenkins.io/job/Infra/job/packer-images/job/main/67/console)

Reply:
EC2 Agents updated:

```text
Running on EC2 (aws) - Ubuntu 18.04 LTS (i-09ffbfde042712eb9) in /home/jenkins/workspace/Infra_packer-images_main
16:08:25  [Pipeline] {
16:08:25  [Pipeline] sh
16:08:26  + mvn -v
16:08:28  Apache Maven 3.8.1 (05c21c65bdfed0f71a2f2ada8b84da59348c4c5d)
16:08:28  Maven home: /usr/share/apache-maven-3.8.1
16:08:28  Java version: 11.0.11, vendor: Ubuntu, runtime: /usr/lib/jvm/java-11-openjdk-amd64
16:08:28  Default locale: en, platform encoding: UTF-8
16:08:28  OS name: "linux", version: "5.4.0-1049-aws", arch: "amd64", family: "unix"
```

```text
16:09:31  Running on EC2 (aws) - High memory ubuntu 18.04  (i-06186849aad247c51) in /home/jenkins/workspace/Infra_packer-images_main
16:09:31  [Pipeline] {
16:09:31  [Pipeline] sh
16:09:33  + mvn -v
16:09:34  Apache Maven 3.8.1 (05c21c65bdfed0f71a2f2ada8b84da59348c4c5d)
16:09:34  Maven home: /usr/share/apache-maven-3.8.1
16:09:34  Java version: 11.0.11, vendor: Ubuntu, runtime: /usr/lib/jvm/java-11-openjdk-amd64
16:09:34  Default locale: en, platform encoding: UTF-8
16:09:34  OS name: "linux", version: "5.4.0-1049-aws", arch: "amd64", family: "unix"
```

```text
16:10:36  Running on EC2 (aws) - ARM64 ubuntu 18.04  (i-01707b50e3de31be8) in /home/jenkins/workspace/Infra_packer-images_main
16:10:36  [Pipeline] {
16:10:36  [Pipeline] sh
16:10:39  + mvn -v
16:10:41  [1mApache Maven 3.8.1 (05c21c65bdfed0f71a2f2ada8b84da59348c4c5d)[m
16:10:41  Maven home: /usr/share/apache-maven-3.8.1
16:10:41  Java version: 11.0.11, vendor: Ubuntu, runtime: /usr/lib/jvm/java-11-openjdk-arm64
16:10:41  Default locale: en, platform encoding: UTF-8
16:10:41  OS name: "linux", version: "5.4.0-1049-aws", arch: "aarch64", family: "unix"
```

```text
16:11:45  Running on EC2 (aws) - Windows 2019 (i-0ff36d22145fe3c12) in C:\Jenkins\workspace\Infra_packer-images_main
16:11:45  [Pipeline] {
16:11:45  [Pipeline] bat
16:11:46  
16:11:46  administrator@EC2AMAZ-54BIHDM C:\Jenkins\workspace\Infra_packer-images_main>mvn -v 
16:11:54  Apache Maven 3.8.1 (05c21c65bdfed0f71a2f2ada8b84da59348c4c5d)
16:11:54  Maven home: c:\tools\apache-maven-3.8.1\bin\..
16:11:54  Java version: 11.0.9, vendor: AdoptOpenJDK, runtime: c:\tools\jdk-11.0.9+11
16:11:54  Default locale: en_US, platform encoding: Cp1252
16:11:54  OS name: "windows server 2019", version: "10.0", arch: "amd64", family: "windows"
```

Reply:
Are azure agents updated?